apps/comps/wagmi-config.ts

```diff
@@ -2,13 +2,14 @@ import { getDefaultConfig } from "connectkit";
 import { Config, createConfig } from "wagmi";
 import { mainnet } from "wagmi/chains";
 
+import { NEXT_PUBLIC_FRONTEND_URL } from "@/config";
+
 export const clientConfig: () => Config = () => {
   const configParams = getDefaultConfig({
     appName: "Recall Competitions",
     appDescription: "Compete with your AI agents in Recall competitions",
-    appUrl:
-      process.env.NEXT_PUBLIC_FRONTEND_URL || "https://app.recall.network",
-    appIcon: `${process.env.NEXT_PUBLIC_FRONTEND_URL}/favicon.ico`,
+    appUrl: NEXT_PUBLIC_FRONTEND_URL || "https://app.recall.network",
+    appIcon: `${NEXT_PUBLIC_FRONTEND_URL}/favicon.ico`,
```

Comment:
fwiw, i think we _technically_ don't need the wagmi config, anymore. it was for SIWE/walletconnect/connectkit. but, maybe we can use the same blobs URL here too

Reply:
ah right. nope. just uploaded it here: https://5pskttgrmgbdllus.public.blob.vercel-storage.com/favicon.ico